NestJS/src/gmail_api/gmailapi.js

```diff
@@ -3,7 +3,7 @@ const readline = require('readline');
 const {google} = require('googleapis');
 var Base64 = require('js-base64').Base64;
 var exports = module.exports = {};
-const dotenv = require('dotenv').config();
+const dotenv = require('dotenv').config({path: "/Users/Mukhamed/Downloads/pro/js-models/test/.env"});
```

Comment:
can you use relative path's instead of absolute?

Reply:
It requires to use relative path.